City-of-Helsinki/drupal-helfi
# UHF-1478 TPR lists

How to test: 
- `composer create-project City-of-Helsinki/drupal-helfi-platform:dev-main hel-platform --no-interaction --repository https://repository.drupal.hel.ninja/`
- `cd hel-platform`
- Switch to corresponding branches: `composer require drupal/helfi_tpr:dev-UHF-1478_tpr_lists && composer require drupal/helfi_platform_config:dev-UHF-1478_tpr_lists`
- `make new`
- `make shell` and `MIGRATE_LIMIT=60 drush mim tpr_unit && MIGRATE_LIMIT=60 drush mim tpr_service`
- Log in to the site and go to Unit list (https://hel-platform.docker.sh/fi/admin/content/integrations/tpr-unit) and Service list (https://hel-platform.docker.sh/fi/admin/content/integrations/tpr-service) and check that the views has been updated to match the content view (https://hel-platform.docker.sh/fi/admin/content).

Also check these:
- https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/86
- https://github.com/City-of-Helsinki/drupal-module-helfi-tpr/pull/18
- https://github.com/City-of-Helsinki/drupal-hdbt-admin/pull/41
